src/layout/Payment/SkeletonLoader/SkeletonLoader.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { Heading, Skeleton } from '@digdir/designsystemet-react';
+
+import classes from 'src/layout/Payment/SkeletonLoader/SkeletonLoader.module.css';
+
+export const SkeletonLoader = () => (
```

Comment:
Nice! Maybe this should replace our own `Loader.tsx`? Or maybe that could be used instead of this? It's not clear to me what's the difference between the two. Anyway, this should possibly not be tied to the payment component, and it might be something we'd want to move to `src/core/loading/`?